ci-operator/step-registry/openshift/microshift/infra/conformance-setup/openshift-microshift-infra-conformance-setup-commands.sh

```diff
@@ -6,9 +6,22 @@ curl https://raw.githubusercontent.com/openshift/release/master/ci-operator/step
 source /tmp/ci-functions.sh
 ci_script_prologue
 
+if "${SRC_FROM_GIT}"; then
```

Comment:
Comment this heavily as it will confuse anyone who sees it outside microshift.

Reply:
It was added as part of the env variable documentation: https://github.com/openshift/release/pull/56264/files#diff-16579d4c09c7af8be7ea934c9b0428bc84d3137cec9d62e19831a40790988ea1R16-R17
Do we need more detail?